sdk/samples/core/README.md

```diff
@@ -25,7 +25,7 @@ The samples' instructions include specifics for Linux based systems. The Linux e
   sudo apt-get install build-essential curl zip unzip tar pkg-config
   ```
 - Have [Git](https://git-scm.com/download) installed.
-<!-- - Have Microsoft [vcpkg](https://github.com/microsoft/vcpkg) package manager and [Mosquitto MQTT C client](TODO) or [Eclipse Paho MQTT C client](https://www.eclipse.org/paho/) installed. This installation may take an extended amount of time (~15-20 minutes).
+- Have Microsoft [vcpkg](https://github.com/microsoft/vcpkg) package manager and [Mosquitto MQTT C client](https://mosquitto.org/download/) or [Eclipse Paho MQTT C client](https://www.eclipse.org/paho/) installed. This installation may take an extended amount of time (~15-20 minutes).
```

Comment:
Given that we have the automatic vcpkg installation now, would it be useful to add instructions on how to use the auto install?

Reply:
Done! Removed explicit vcpkg instructions and added a link to learn more about manually setting these options.